ui/ducks/send/send.js

```diff
@@ -1250,15 +1253,34 @@ const slice = createSlice({
      *
      * @param {SendStateDraft} state - A writable draft of the send state to be
      *  updated.
+     * @param {GasEstimateUpdatePayload} action - The gas fee update payload
```

Comment:
Place `[]` around 'action' because its optional. Only one callsite passes an action payload.

Reply:
Will fix that